Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,63 @@
     "\n",
-    "# Handy Commands for Writing the Physics file: \n",
+    "Once the solution has attained a steady like solution, next step is to look at the results. Start with mean values of Cl and Cd at differernt $\\alpha$ as shown below.\n",
     "\n",
-    "## Insert a Figure\n",
+    "<img src=\"./Images/ClCdvsAlpha.png\" width=\"900\">\n",
+    "Fig. 2. Variation of Cd and Cl with $\\alpha$.\n",
     "\n",
-    "```\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
-    "</br>\n",
-    "Fig. 1 - The caption for the image goes here.\n",
-    "```\n",
+    "As $\\alpha$ increases, Cd decreases. The pressure drag decreases with increase in skin friction drag, resulting in a net decrease in total drag force. As $\\alpha$ increases, Cl increases (look at the values on the y axis). This is due to Magnus effect produced by rotation of the cylinder. Due to counter-clockwise rotation, the streamlines stack at the bottom of the cylinder creating a low pressure region there. The pressure at the top is high and this pressure difference gives rise to a negative lift. As the roational velocity increases, the pressure difference increases and so does the lift force. \n",
     "\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
-    "</br>\n",
-    "Fig. 1 - The caption for the image goes here."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Insert a YouTube video Available on the SFO Channel \n",
+    "Unsteady flow over cylinder gives rise to alternating vortex shedding behind the cylinder. This vortex shedding has a shedding frequency. A non dimensionl number, Strouhal number (St) is associated with this frequency. Strouhal number is defined as $St = D\\omega/U_\\infty$, where D is the diameter of the cylinder. When $D = 1$ and $U_\\infty =1$, St is simply $\\omega$ as in this case. Strouhal number is atio of the unsteadiness in the flow to inertial forces in the flow field. Calulate the value of St for different vlaues of $\\alpha$ and compare with other experimental and numerical results. Here the values are compared with the results of Kang et al (2006)[3] & Mittal and Kumar (2003)[4] (shown to compare the trend of St with α). The results seem to be in good agreement with each other.\n",
```

Comment:
The description of Strouhal number is very precise and good. However, regarding the comparison between the results, what do you mean by "good agreement"?

-I think it is a good idea to increase the scale of the Y-axis. 
-Also provide a detail explanation for the differences between two case study and CFD.
-We should also define why CFD results don't show a constant trend like other results? Furthermore, why CFD is underestimating values?